app/build.gradle

```diff
@@ -106,8 +106,8 @@ dependencies {
     implementation "com.squareup.retrofit2:retrofit:$retrofitVersion"
     implementation "com.squareup.retrofit2:converter-gson:$retrofitVersion"
     implementation "com.squareup.retrofit2:adapter-rxjava2:$retrofitVersion"
-    implementation 'com.squareup.okhttp3:okhttp:3.14.2'
-    implementation 'com.squareup.okhttp3:logging-interceptor:3.14.2'
+    implementation 'com.squareup.okhttp3:okhttp:4.0.1'
```

Comment:
There are proguard issues:
https://github.com/square/okhttp/issues/5167
do we use R8 already?

Reply:
Yes we use R8. I will check if everything works, thx